README.md

```diff
@@ -29,104 +23,79 @@ Beehive requires Go 1.7 or higher. Make sure you have a working Go environment.
+You can build and install the `beehive` binary like other Go binaries out there (`go get -u`) but you'll need to make sure beehive can find the assets (images, javascript, css, etc). See the Troubleshooting/Notes section for additional details.
 
 Run `beehive --help` to see a full list of options.
 
 ### Deployment Tools
- - [Dockerfile](docker)
- - [Ansible](https://github.com/morbidick/ansible-role-beehive)
+
+- [Dockerfile](docker)
+- [Ansible](https://github.com/morbidick/ansible-role-beehive)
 
 ## Configuration
 
-Think of Hives as little plugins, extending Beehive's abilities with events you
-can react on and actions you can execute.
+Think of Hives as little plugins, extending Beehive's abilities with events you can react on and actions you can execute.
 
 Just as examples, there's a Twitter plugin that can
- - react to someone you follow posting a tweet (an event)
- - post a new tweet for you (an action)
- - ...
+
+- react to someone you follow posting a tweet (an event)
+- post a new tweet for you (an action)
+- ...
 
 or an RSS plugin that lets you
- - monitor RSS feeds and react on new feed items (another event)
+
+- monitor RSS feeds and react on new feed items (another event)
 
 or an email plugin that gives you the ability to
- - send emails (another action)
 
-Each Hive lets you spawn one or multiple Bees in it, all working independently
-from another. That allows you to create separate plugin instances, e.g. one
-email-Bee for your private mail account, and another one for your work email.
+- send emails (another action)
+
+Each Hive lets you spawn one or multiple Bees in it, all working independently from another. That allows you to create separate plugin instances, e.g. one email-Bee for your private mail account, and another one for your work email.
 
 ### Creating Bees
 
-Sounds complicated? It's not! Just for fun, let's setup Beehive to send us an
-email whenever an RSS feed gets updated. Start `beehive` and open <http://localhost:8181/>
-in your browser. Note that Beehive will create a config file `beehive.conf`
-in its current working directory, unless you specify a different file with the
-`-config` option.
+Sounds complicated? It's not! Just for fun, let's setup Beehive to send us an email whenever an RSS feed gets updated. Start `beehive` and open <http://localhost:8181/> in your browser. Note that Beehive will create a config file `beehive.conf` in its current working directory, unless you specify a different file with the `-config` option.
 
-Note: You currently have to start `beehive` from within $GOPATH/src/github.com/muesli/beehive
-in order for it to find all the resources for the admin interface. Also see the
-Troubleshooting & Notes section of this README.
+Note: You currently have to start `beehive` from within $GOPATH/src/github.com/muesli/beehive in order for it to find all the resources for the admin interface. Also see the Troubleshooting & Notes section of this README.
```

Comment:
Wrap the `$GOPATH/...` line in backticks. Add a link to the _Toubleshooting & Notes_ section, using an anchor or otherwise.

Reply:
Could be more of these. I’ll make another pass through.